src/expr/impl/src/scalar/ai_model.rs

```diff
@@ -173,55 +209,39 @@ impl Expression for OpenAiEmbedding {
     }
 }
 
-#[build_function("openai_embedding(varchar, varchar, varchar) -> float4[]")]
```

Comment:
The old syntax has already been released. Will this be a breaking change?

Reply:
Yes, it will be a breaking change. I've included the breaking change in the release note section. I think the feature is still experimental, so no need to handle backward compatibility yet.